src/overlays/actors/ovl_En_Horse/z_en_horse.c

```diff
@@ -1,19 +1,68 @@
 #include "z_en_horse.h"
+#include "overlays/actors/ovl_En_In/z_en_in.h"
 
 #define FLAGS 0x00000010
 
 #define THIS ((EnHorse*)thisx)
 
+typedef void (*EnHorseCsFunc)(EnHorse*, GlobalContext*, CsCmdActorAction*);
+typedef void (*EnHorseActionFunc)(EnHorse*, GlobalContext*);
+
+// z_horse.c
+void func_8006DD9C(Actor* actor, Vec3f* arg1, s16 arg2);
+
+void func_80028A54(GlobalContext* globalCtx, f32 arg1, Vec3f* vec);
```

Comment:
if these dont exist in functions.h already they can go there instead of declared here